updating/understanding-openshift-updates.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Understanding {product-title} updates
+include::modules/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update a {product-title} cluster with a single click using over the air update.
+Platform administrators are automatically notified when an update is available for their cluster.
+Updates can be applied either from the `oc` command-line interface (CLI) or the web console.
+
+The {product-title} update service builds a graph of upgrade possibilities based on release images in the registry.
+This graph is based on allowable tested update paths from a specific version.
+{product-title} clusters connect to the Red Hat Hybrid Cloud servers and identify which clusters the user is running along with version information.
+Policy engines combine the information from customer entitlement in conjunction with the update graph to tell the cluster which release they can upgrade to.
```

Comment:
I think "policy engines" is getting into internal implementation details for the update service.  Those are useful details when we're talking to folks who are about to run their own local update service.  But this doc seems to be aimed more at the admin thinking about updating, in which case the detail may be distracting.  Maybe `The OpenShift Update Service responds with information about known update targets.` or some such, because now that we have conditional updates it's broader than `...about recommended updates`, and if someone is using a `candidate-4.*` channel, it's broader than `...about supported updates`.